context/AuthContext.js

```diff
@@ -0,0 +1,71 @@
+import React,{createContext,useEffect,useState} from 'react';
+import axios from 'axios';
+import { BASE_URL } from '@env';
+import {LoginEndpoint,SignupEndpoint,UserFile,TokenFile} from '../APIs.js'
+// import EncryptedStorage from 'react-native-encrypted-storage';
+import {Storage} from 'expo-storage'
+import * as FileSystem from 'expo-file-system';
+
+export const AuthContext = createContext();
+
+export const AuthProvider = ({children}) =>{
+    const [isLoading,setIsLoading]=useState(false)
+    const [userToken,setUserToken]=useState(null)
+    const [user,setUser]=useState(null)
+    const login = async (data) =>{
+        setIsLoading(true)
+        try{
+            const response = await axios.post(`${BASE_URL}${LoginEndpoint}`,data)
```

Comment:
We can configure axios to have a default base url by adding `axios.defaults.baseURL =` in `index.js` so we don't have to include it every time we call our API